client/scss/components/_chooser.scss

```diff
@@ -1,131 +1,92 @@
-/*
-TODO this chooser style has been made more generic based on two identical
-methods for choosing pages and images that were previously included in their
-own less files in each app directory (and since deleted). It would be best if
-an admin 'theme' provided all the design for a UI in a single place, but
-should that be a series of overrides to the css provided from an app? If so,
-perhaps those two previous less files should be re-instated and then
-overridden here? hmm.
-*/
-
 .chooser {
```

Comment:
`w-chooser` ? 😀

Reply:
I’ve tried to leave refactorings to a minimum for this one, we’re way past schedule 🙈